pkgs/top-level/dhall-packages.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, newScope
+, overrides ? (self: super: {})
+}:
+
+let
+  packages = self:
+    let
+      callPackage = newScope self;
+
+      buildDhallPackage =
+        callPackage ../development/interpreters/dhall/build-dhall-package.nix { };
+
+    in
+      { inherit buildDhallPackage;
+
+        dhall-kubernetes =
+          callPackage ../development/dhall-modules/dhall-kubernetes.nix { };
+
+        dhall-packages =
+          callPackage ../development/dhall-modules/dhall-packages.nix { };
```

Comment:
what’s the reasoning behind this attribute? It doesn’t seem to contain anything that the others don’t expose.

Reply:
`dhall-packages` does not just re-export other Dhall packages.  `dhall-packages` provides new functionality of its own (like Argo CD bindings, for example)

Reply:
Oh, I see, `dhall-packages` is a (mono)repository containing other dhall packages.